spec/defines/vhost_spec.rb

```diff
@@ -154,6 +154,7 @@
           'ssl_verify_depth'            => '3',
           'ssl_options'                 => '+ExportCertData',
           'ssl_openssl_conf_cmd'        => 'DHParameters "foo.pem"',
+          'ssl_proxy_verify'            => 'require',
```

Comment:
Oops sorry @occelebi . I misread the diff and didn't see this was in the test and not manifest. This is fine the way it is! Thanks!